dashboard/src/components/Breadcrumb.vue

```diff
@@ -32,7 +35,7 @@
   </div>
 </template>
 <script setup>
-import { defineProps, ref } from 'vue'
```

Comment:
No idea really, but a quick search came up with this: https://vueschool.io/articles/vuejs-tutorials/eslint-and-prettier-with-vite-and-vue-js-3/

Reply:
please create an issue for this before merging the PR. this might be happening in other places in the codebase so setting up a linter/prettier sooner rather than later should be good.

Reply:
https://github.com/fossunited/fossunited/issues/578